hs_core/discovery_parser.py

```diff
@@ -111,7 +111,7 @@ class ParseSQ(object):
         'modified',
         'organization',
         'publisher',
-        'rating',
+        # deprecated # 'rating',
```

Comment:
I prefer to delete these instead of turning them into a comment with a deprecated note.  I can't think of a reason why keeping them around in comments would be useful.